src/kOS/Module/kOSProcessor.cs

```diff
@@ -201,7 +199,9 @@ public IScreenBuffer GetScreen()
         // TODO - later refactor making this kOS.Safer so it can work on ITermWindow, which also means moving all of UserIO's classes too.
         public Screen.TermWindow GetWindow()
         {
-            return shared.Window;
+            if (shared.Window != null)
+                return shared.Window;
+            return GetComponent<Screen.TermWindow>();
```

Comment:
In testing I had a single instance where the new shared `DestroyObjects` had been called before the last attempt to call `GetWindow`, but it is possible that later changes removed the error case.  The problem with an error that you only see a few times is that it's hard to say for sure that you eliminated every chance for it to appear.  Especially since we don't control when Unity calls various methods.